cgi-bin/LJ/ConfCheck/General.pm

```diff
@@ -759,7 +758,6 @@ my %bools = (
              "OPENID_COMPAT" => "Support pre-1.0 OpenID specs as well as final spec.",
              "OPENID_STATELESS" => "Speak stateless OpenID.  Slower, but no local state needs to be kept.",
              "ONLY_USER_VHOSTS" => "Don't allow www.* journals at /users/ and /~ and /community/.  Only allow them on their own user virtual host domains.",
-             "USERPIC_MOGILEFS" => "Store userpics on MogileFS.",
```

Comment:
You remove the USERPIC_MOGILEFS setting here, and the mogfs_userpic_key user method below, but those are both still required by bin/upgrading/migrate-userpics.pl. Should we remove that script?  (It applies to any rows in userpic2 located somewhere other than mogile - given that you've added blobstore as a new possible value in that column, I'm guessing it should go away.)